Kajabi/sage-archived
# Revised font loading

## Description
Removes remote dependency for "Inter" webfont. Font files were added to local assets, replaced external CSS and prioritized local font sources.

|  Before   |  After  |
|--------|--------|
|![webfont-before](https://user-images.githubusercontent.com/816579/74202600-befb6680-4c21-11ea-93ae-af906336ee4f.png)|![webfont-after](https://user-images.githubusercontent.com/816579/74202604-c3c01a80-4c21-11ea-8685-60aa840f3b26.png)|

## Test notes
1. Spin up a local server with `rails s`
2. Open the developer tools in your browser, and navigate to the "Network" tab
3. Verify that the Inter font file(s) and CSS are no longer loaded externally - the request URL should be going to `localhost`

## Related issues
Resolves #25.
